packages/icons/src/utilities/downloadSVG.ts

```diff
@@ -20,17 +20,12 @@ export const downloadSVG = async (icon: IIcon, output: string) => {
 
   try {
     const svg = await fetch(icon.meta.downloadLink, getRequestInit());
-    
     const svgString = await svg.text();
     const optimizedSvg = optimize(extractSVG(svgString), svgoOptions);
-    
-    if (optimizedSvg.error) {
-      throw new Error(optimizedSvg.error);
-    }
```

Comment:
Why was this throw removed?

Reply:
This doesn't exist anymore. SVGO should throw errors directly since v3 (untested), source: https://github.com/svg/svgo/issues/1429